tests/testthat/test-parsnip-case-weights.R

```diff
@@ -0,0 +1,1143 @@
+# theses are needed for all case-weights tests
+skip_if_not_installed("parsnip", "0.2.1.9001")
+skip_if_not_installed("hardhat", "0.2.0.9000")
+skip_if_not_installed("yardstick", "0.0.9.9000")
+skip_if_not_installed("workflows", "0.2.6.9001")
+skip_if_not_installed("recipes", "0.2.0.9001")
```

Comment:
Shouldn't these be on `1.0.0`?

Reply:
I left it originally because it should also work with those old dev versions. would you change it because it's easier to read?

Reply:
In my mind, it should work for all parts of `1.0.0`, but it wouldn't work for all parts of `0.2.0.9000`. I don't think it matter too much since it is unlikely that those old versions will be installed

Reply:
I'll change them to 1.0.0